dluhc-component-library/src/components/maps/DatasetLayer.tsx

```diff
@@ -20,13 +23,29 @@ const DatasetLayer = ({ area, dataset, zIndex = 1 }: DatasetlayerProps) => {
     return null;
   }
 
+  const stroke: StrokeOptions = useMemo(() => {
+    const color = dataset["paint-options"]
+      ? dataset["paint-options"].colour
+      : "#003078";
+    return { color, width: 2 };
+  }, [dataset]);
+
+  const fill: FillOptions = useMemo(() => {
+    const paintOptions = dataset["paint-options"];
+    if (!paintOptions) {
+      return { color: "rgba(0, 48, 120, 0.2)" };
+    }
+
+    const { colour, opacity } = paintOptions;
+    const red = parseInt(colour.slice(1, 3), 16);
```

Comment:
`Non-blocker` Guessing there is no colour parser?

Reply:
Nothing by default in JavaScript that I could see, there'll be libraries out there, but I just though adding a function ourselves would be simple enough.